docs/guide/install-and-run.asciidoc

```diff
@@ -1,113 +1,141 @@
+
+* {apm-server-ref-v}/configuring-howto-apm-server.html[Configuring APM Server]
+** {apm-server-ref-v}/configuration-process.html[General configuration options]
+** {apm-server-ref-v}/elasticsearch-output.html[Configure the Elasticsearch output]
 
 [[secure-api-access]]
-[float]
-==== Secure access to the API
-If you change the listen address from `localhost` to something that is accessible from outside of the machine,
+If you do change the listen address from `localhost` to something that is accessible from outside of the machine,
 we recommend setting up firewall rules to ensure that only your own systems can access the API.
 Alternatively,
 you can use a {apm-server-ref}/securing-apm-server.html[secret token and TLS].
 
-If you have APM Server running on the same host as your service, you can configure it to listen on a Unix domain socket.
-
-[[kibana-dashboards]]
-[float]
-==== Install the Kibana dashboards
-
-From APM Server and Kibana 6.4 on you can load dashboards directly via the {kibana-ref}/apm-getting-started.html[APM 
-Kibana UI].
-
-See an example screenshot of a Kibana dashboard:
-
-image::kibana-dashboard.png[Screenshot of a Kibana Dashboard]
+If you have APM Server running on the same host as your service,
+you can configure it to listen on a Unix domain socket.
 
 [[more-information]]
-[float]
-=== More information
-For detailed instructions on how to install and secure APM Server in your server environment,
+TIP: For detailed instructions on how to install and secure APM Server in your server environment,
 including details on how to run APM Server in a highly available environment,
-please see {apm-server-ref}/index.html[APM Server documentation].
+please see the full {apm-server-ref-v}/index.html[APM Server documentation].
 
-Once APM Server is up and running,
-you need to install an agent in your service.
+[[kibana-dashboards]]
+[float]
+=== Install the Kibana dashboards
+
+Load APM dashboards directly in Kibana via the {kibana-ref}/apm-getting-started.html[APM - Getting started] page.
 
 [[agents]]
 [float]
 === Install and configure APM agents
 
 Agents are written in the same language as your service.
-Currently Elastic APM has agents for Node.js, Python, Ruby, Java, Go, and JavaScript RUM.
+Currently Elastic APM has agents for Go, Java, Node.js, Python, Ruby, and JavaScript RUM.
 
+// todo: fix this sentence
 Setting up a new service to be monitored requires installing the agent,
 and configuring it with the address of your APM Server and the service name.
 
+NOTE: Be sure to check the <<agent-server-compatibility,Agent/Server compatibility matrix>> to ensure you're using agents that are compatible with your version of Elasticsearch
+
+[cols="h,,,"]
+|=======================================================================
+|Agent
+3+| More information
+
+.2+|Go
+3+|The Go agent automatically instruments Gorilla and Gin, as well as support for Go’s built-in net/http and database/sql drivers.
```

Comment:
is `as well as support` grammatically correct? i was expecting `adds support` or `supports`